frontend/packages/core/src/AppLayout/notifications.tsx

```diff
@@ -73,14 +115,26 @@ const Notifications: React.FC = () => {
                   autoFocusItem={open}
                   id="notification-options"
                   onKeyDown={handleListKeyDown}
-                />
+                >
+                  {data?.map(d => {
+                    return (
+                      <MenuItem>
+                        <ListItemText>{d.value}</ListItemText>
+                      </MenuItem>
+                    );
+                  })}
+                </MenuList>
               </ClickAwayListener>
             </Paper>
           </Grow>
         )}
       </Popper>
-    </Box>
+    </>
   );
 };
 
+const Notifications: React.FC<NotficationsProp> = ({ disabled = true, data }) => {
+  return <>{disabled ? null : <UserNotifications data={data} />}</>;
```

Comment:
is the fragment necessary? i think it can be done without

Reply:
anyways on second thought i think i'd leave it up to the parent to render it or not, it's a bit weird to do it in the component itself IMO. also disabled on a component generally implies a disabled state not a null render.